src/PaymentGateways/PayPalCommerce/AdminSettingFields.php

```diff
@@ -117,99 +112,27 @@ public function payPalCommerceAccountManagerField()
-						<span class="give-field-description">
-							<i class="fa fa-check"></i>
-							<?php
-                            printf(
-                                '%1$s <span class="paypal-account-email">%2$s</span>',
-                                esc_html__('Connected for payments as', 'give'),
-                                give(MerchantDetail::class)->merchantId
-                            );
-                            ?>
-						</span>
-                                <span class="actions">
-							<button
-                               id="js-give-paypal-disconnect-paypal-account"><?php
-                                    esc_html_e('Disconnect', 'give'); ?></button>
-						</span>
-                            </div>
-                            <div class="api-access-feature-list-wrap">
-                                <p><?php
-                                    esc_html_e('APIs Connected:', 'give'); ?></p>
-                                <ul>
-                                    <li><?php
-                                        esc_html_e('Payments', 'give'); ?></li>
-                                    <?php
-                                    if ($isRecurringAddonActive) : ?>
-                                        <li><?php
-                                            esc_html_e('Subscriptions', 'give'); ?></li>
-                                    <?php
-                                    endif; ?>
-                                    <li><?php
-                                        esc_html_e('Refunds', 'give'); ?></li>
-                                </ul>
-                            </div>
-                        </div>
-                        <?php
-                        $this->printErrors(); ?>
-                    </div>
-                </div>
-            </td>
-        </tr>
-        <?php
+
+        // Show live PayPal connect button.
+        $paypalLiveSetting = new \stdClass();
+        $paypalLiveSetting->label = esc_html__('PayPal Connection', 'give');
+        $paypalLiveSetting->mode = 'live';
+        $paypalLiveSetting->connectButtonLabel = esc_html__('Connect with PayPal Live', 'give');
+        $paypalLiveSetting->description = esc_html__('PayPal is currently NOT connected.', 'give');
+        $paypalLiveSetting->isRecurringAddonActive = $isRecurringAddonActive;
+        echo $this->getPayPalConnectionSettingView($paypalLiveSetting);
+
+        // Show sandbox PayPal connect button.
+        $paypalSandboxSetting = new \stdClass();
+        $paypalSandboxSetting->label = esc_html__('PayPal Sandbox Connection', 'give');
+        $paypalSandboxSetting->mode = 'sandbox';
+        $paypalSandboxSetting->connectButtonLabel = esc_html__('Connect with PayPal Sandbox', 'give');
+        $paypalSandboxSetting->description = esc_html__(
+            'PayPal sandbox is currently NOT connected. This is a separate sandbox PayPal account used for testing. Live PayPal accounts will not work.',
```

Comment:
In addition to wording, please have "PayPal Sandbox account" link to https://www.sandbox.paypal.com
```suggestion
            'PayPal sandbox is currently NOT connected. This is a separate PayPal Sandbox account, used for testing. Live PayPal accounts will not work.',
```